.github/workflows/acceptance-py3.6.yml

```diff
@@ -0,0 +1,109 @@
+name: acceptance
+on:
+  push:
+    branches:
+      - master
+      - releases/**
+  pull_request:
+
+jobs:
+  # TODO(billy): Currently, keeping name the same because of `required` status checks
+  py3-acceptance:
+    name: python3.6 acceptance
+    runs-on: ubuntu-16.04
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        instance: [0, 1, 2]
+
+    env:
+      MIGRATIONS_TEST_MIGRATE: 1
+      VISUAL_SNAPSHOT_ENABLE: 1
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: volta-cli/action@v1
+
+      # If we make these jobs "required" to merge on GH, then on every PR, GitHub automatically
+      # creates a status check in the "pending" state. This means that the workflow needs to run
+      # for every PR in order to update the status checks.
+      #
+      # In order to optimize CI usage, we want the tests to only run when python files change,
+      # since frontend changes should have no effect on these test suites. We cannot use GH workflow
+      # path filters because entire workflow would be skipped vs skipping individual jobs which
+      # would still allow this status check to pass.
+      - name: Check for python file changes
+        uses: getsentry/paths-filter@v2
+        id: changes
+        with:
+          token: ${{ github.token }}
+          filters: .github/file-filters.yml
+      # XXX: If taking snapshots with this, be sure to remove above and the following `if` conditions!
```

Comment:
Could we disable snapshotting entirely by unsetting VISUAL_SNAPSHOT_ENABLE since we're not running visual-diff yet?

Or, why not just start snapshotting py3 as well? I tried in https://github.com/getsentry/sentry/pull/21186#issuecomment-704641754 but gave up trying to make py3 snapshots the same as py2. So easiest way would just filename suffix py3 snapshots, not sure how much work that would take with the way visual-diff works though.

Reply:
Yeah we can unset the env var -- the visual diff platform is not yet robust enough to support multiple projects, but we may be able to make that happen later on

Reply:
Oh, that sounds like a blocker, unless we want to switch from py2 to py3 snapshots entirely. cc @getsentry/python-apocalypse

Reply:
I can add support if it's a blocker